packages/api/cms-api/src/builds/builds.service.ts

```diff
@@ -70,15 +76,26 @@ export class BuildsService {
         return true;
     }
 
-    async getBuilds(options?: { limit?: number | undefined }): Promise<BuildObject[]> {
+    /**
+     * Should only be used internally
+     */
+    async dangerouslyCreateBuildsWithoutUserCheck(trigger: string): Promise<boolean> {
+        const builderCronJobs = await this.kubernetesService.getAllCronJobs(
+            `${BUILDER_LABEL} = true, ${INSTANCE_LABEL} = ${this.kubernetesService.helmRelease}`,
+        );
+        return this.createBuilds(trigger, builderCronJobs);
+    }
+
+    async createBuildsWithUserCheck(trigger: string, builderCronJobs: V1CronJob[]): Promise<boolean> {
```

Comment:
I don't understand this: what about the user is checked here? 

And why is this user check needed at all? (in addition to the .canAccessScope check in createBuilds)

Reply:
@nsams you are absolutely right. The check, which is not there, is not needed. I've fixed the name.